universal-login-commons/lib/core/utils/ens.ts

```diff
@@ -7,3 +7,6 @@ export const ensNameElementRegex = /^[a-z0-9](-*[a-z0-9]+)*$/;
 export const isValidEnsNameElement = (ensNameElement: string) => {
   return ensNameElementRegex.test(ensNameElement);
 };
+
+export const isValidEnsName = (ensName: string) =>
+  !!ensName.match(/^[a-z0-9]*(-*[a-z0-9]+)\.[a-z0-9](-*[a-z0-9]+)\.(eth|xyz|test)/);
```

Comment:
I think we will have many more ens tld's than just eth, xyz & test.

Read more [here](https://medium.com/the-ethereum-name-service/upcoming-changes-to-the-ens-root-a1b78fd52b38)